### PR TITLE
Fast collection charts

### PR DESCRIPTION
**Pull request by vikiival**
What's new:

I rewrote the chart resolvers so the data are aggregated by days, that means we have cleaner chart and also the app is super fast (donkey collection loads 1s vs 67s before).

- :construction: better graph query
- :zap: added function to convert from and to decimals
- :zap: utils to manipulate with collection chart
- :zap: mapDecimals
- :top: Squid @ 005
- :alien: graphql query for manipulating with collection chart data
- :zap: rewrite logic for handling chart data
- :zap: CollectionPriceChart now reflects the changes
- :zap: added line for median price
- :zap: return count to the chart

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [x] Feature
- [x] Refactoring

### What's new?

- [ ] PR closes #<issue_number>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/0a8ce195286c168f19-DONKEY?tab=activity>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

**BEFORE**
Screenshot 2022-02-20 at 22 34 33

**AFTER**
Screenshot 2022-02-20 at 22 45 25

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default {
       ...defineApolloConfig(),
       subsquid: toApolloEndpoint(
         process.env.SUBSQUID_ENDPOINT ||
-          'https://app.gc.subsquid.io/beta/rubick/004/graphql'
+          'https://app.gc.subsquid.io/beta/rubick/005/graphql'
```
Comment (roiLeo) [inline nuxt.config.js:251]: new endpoint?

File: components/rmrk/Gallery/CollectionItem.vue [hunk 2]
```diff
@@ -111,7 +111,9 @@ import {
 import isShareMode from '@/utils/isShareMode'
 import shouldUpdate from '@/utils/shouldUpdate'
 import collectionById from '@/queries/collectionById.graphql'
-import nftListByCollection from '@/queries/nftListByCollection.graphql'
+import collectionNftEventListById from '@/queries/collectionNftEventListById.graphql'
+import collectionChartById from '@/queries/rmrk/subsquid/collectionChartById.graphql'
+// collectionNftEventListById
```
Comment (roiLeo) [inline components/rmrk/Gallery/CollectionItem.vue:116]:
```suggestion
```

**Review by roiLeo — COMMENTED** [review]
code lgtm 
beware some of us still have old subsquid endpoint in env

**Comment by vikiival** [comment]
OK find out that I'm using average instead of moving avg

**Comment by yangwao** [comment]
let me know when it's fixed tho

**Comment by vikiival** [comment]
@yangwao ready to fly ✈️

**Comment by yangwao** [comment]
ooor not? 

https://deploy-preview-2411--koda-nuxt.netlify.app/rmrk/collection/102cb9e9988c85201e-ZEITGEIST-JUSTICE-TAROT-COLLECTION?tab=activity

image

**Comment by yangwao** [comment]
ah, 004

**Comment by yangwao** [comment]
damn that's superior fast

**Comment by yangwao** [comment]
https://fontawesome.com/icons/chart-candlestick?s=solid so we can directly link from series to graphs :)

**Comment by yangwao** [comment]
Strange no errors 

image

**Comment by yangwao** [comment]
removed env from cf as well

**Comment by yangwao** [comment]
oh it's possible that uses old subsquid?

**Comment by vikiival** [comment]
Draft, because there is a bug in 005

**Pull request merged**